pkg/deploy/service_test.go

```diff
@@ -57,23 +59,26 @@ func (suite *ServiceTestSuite) TestCreateDeploymentRequestSuccess() {
 		},
 	}
 	for _, c := range cases {
-		received, err := createDeploymentForTests(suite, c.input)
-		if err != nil {
-			suite.T().Fatal(err)
-		}
-		expectedJsonStr, err := ioutil.ReadFile(c.output)
-		if err != nil {
-			suite.T().Fatalf("TestCreateDeploymentRequestSuccess failed with: Error loading tesdata file %s", err)
-		}
-
-		expectedReq := de.PipelineStartPipelineRequest{}
-		err = json.Unmarshal(expectedJsonStr, &expectedReq)
-		if err != nil {
-			suite.T().Fatalf("TestCreateDeploymentRequestSuccess failed with: Error Unmarshalling JSON string to Request obj %s", err)
-		}
-		diffOfExpectedAndRecieved, err := diff.Diff(expectedReq, *received)
-		suite.NoError(err)
-		suite.Len(diffOfExpectedAndRecieved, 0)
+		t.Run(fmt.Sprintf("%s -> %s", c.input, c.output), func(t *testing.T) {
```

Comment:
If you use t.Run you get per-test error messages. Also you can run tests individually in an IDE:

<img width="811" alt="Screen Shot 2022-04-12 at 5 42 25 PM" src="https://user-images.githubusercontent.com/13868700/163060309-03316e9a-5567-4e3b-a11a-79d71e553630.png">